server/src/main/resources/org/opensearch/bootstrap/security.policy

```diff
@@ -85,6 +85,11 @@ grant codeBase "${codebase.zstd-jni}" {
   permission java.lang.RuntimePermission "loadLibrary.*";
 };
 
+// repository-azure plugin and server side streaming
+grant codeBase "${codebase.reactor-core}" {
```

Comment:
@chengwushi-netapp this was the cause of the security exceptions: the `IdentityClient` was run in Reactor pipeline but the `reactor-core` dependency is coming from OpenSearch core, not a Azure repository plugin